lib/components/jumbotron.vue

```diff
@@ -16,6 +16,10 @@
                 type: Boolean,
                 default: false
             },
+            containerFluid: {
```

Comment:
There is already another `fluid` prop for the main jumbotron div though.

The jumbotron itself can be fluid (`jumbotron-fluid`), and the container inside can also be fluid (`continer-fluid`).